Jenkinsfile

```diff
@@ -38,7 +38,8 @@ pipeline {
                     }
                     steps {
                         checkout scm
-                        sh "make build"
+                        sh 'make tests && make build'
```

Comment:
Don't you want to `make build` first?

Reply:
I thought it make sense to build only if the tests to build only if the tests are successful. Imagine you checkout these Dockerfiles, you build them (you build and cache locally a `docker-ssl-slave:latest`) but after your tests fail. Then your Docker cache would have a non working `docker-ssl-slave:latest` image.
But yes, maybe for Jenkins it doesn't make much sense to run the tests if it cannot build anyway (even though the tests **also check** if it can build :D ).
I will revert.